lib/router/router.js

```diff
@@ -109,6 +126,17 @@ p.getInfo = function(req) {
     }
   }
 
+  // forward original image if no operation on image and browser support image format
+  if (routeInfo.imageSteps.length === 0 && (this.options.noop || routeInfo.urlInfo.query.noop)) {
```

Comment:
Pretty sure all of this could be removed if a new `format` option of `same` was added. That way the responding image would only be changed in format if the requestor does not support the image. 

Then you would just set your default options to include `format: "same"`